api/v1alpha1/constant.go

```diff
@@ -19,6 +19,12 @@ const (
 	// The cache system are ready
 	DatasetReadyReason = "DatasetReady"
 
+	// The cache system are update ready
+	DatasetUpdateReadyReason = "DatasetUpdateReady"
```

Comment:
What's the difference between DatasetUpdateReady and DatasetReady?

Reply:
I think we can call them "DatasetReady" together, like "NodeReady" in k8s